doc_source/CNAMEs.md

```diff
@@ -150,11 +152,17 @@ You can’t add a wildcard to a top level domain name, such as `*.com`, so if yo
    ```
 
    The line in the Answer Section shows a CNAME resource record set that routes queries for images\.example\.com to the CloudFront distribution domain name d111111abcdef8\.cloudfront\.net\. The CNAME resource record set is configured correctly if the name on the right side of `CNAME` is the domain name for your CloudFront distribution\. If that is any other value, for example, the domain name for your Amazon S3 bucket, then the CNAME resource record set is configured incorrectly\. In that case, go back to Step 4 and correct the CNAME record to point to the domain name for your distribution\.
+**Note**  
+When using Route 53 alias records, it is not possible to use dig to confirm that the resource record points to the new distribution\. In this case, you can either change the type of of the resource record from alias to CNAME, or wait until the record's time to live \(TTL\) value has expired\.
```

Comment:
There's a typo here: "... you can either change the type of of the resource record ..." (double "of").

Reply:
Addressed with fixup commit, that will auto-squash correctly with `git rebase --autosquash <ref>`